src/dash-adapter.js

```diff
@@ -202,19 +210,29 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     }
   }
 
+
+  _createAdapterFunctionBindings(): void{
+
+      this._adapterEventsBindings['error'] =  this._onError.bind(this);
```

Comment:
can we use arrow function?

Reply:
also, need to clear these bounded methods on destroy or we will have memory leaks possibly